bin/plugin/commands/since.js

```diff
@@ -38,13 +45,39 @@ exports.handler = async ( opt ) => {
 		return;
 	}
 
-	const patterns = [
-		path.resolve( __dirname, '../../../**/*.php' ),
-		path.resolve( __dirname, '../../../**/*.js' ),
-	];
+	if (
+		opt.plugin !== 'performance-lab' &&
+		! plugins.includes( opt.plugin )
+	) {
+		log(
+			formats.error(
+				`The plugin "${ opt.plugin }" is not a valid plugin managed as part of this project.`
+			)
+		);
+		return;
```

Comment:
How about exiting with an error code?

```suggestion
		process.exit(1);
```

Reply:
@thelovekesh @westonruter This actually shouldn't exit or output the error at all. This is handled centrally by the CLI tool, and changing it here unnecessarily couples error handling to the output. Let's change this to simply `throw new Error( ... )`.

Reply:
@felixarntz I think it simply exits when an error is thrown. 

<details><summary>See Output</summary>

```
Error: The plugin "foo" is not a valid plugin managed as part of this project.
    at exports.handler (/home/thelovekesh/Desktop/work/google/wpdev/public/content/plugins/performance/bin/plugin/commands/since.js:61:9)
    at Command.<anonymous> (/home/thelovekesh/Desktop/work/google/wpdev/public/content/plugins/performance/bin/plugin/cli.js:18:10)
    at Command.listener [as _actionHandler] (/home/thelovekesh/Desktop/work/google/wpdev/public/content/plugins/performance/node_modules/commander/lib/command.js:482:17)
    at /home/thelovekesh/Desktop/work/google/wpdev/public/content/plugins/performance/node_modules/commander/lib/command.js:1283:65
    at Command._chainOrCall (/home/thelovekesh/Desktop/work/google/wpdev/public/content/plugins/performance/node_modules/commander/lib/command.js:1177:12)
    at Command._parseCommand (/home/thelovekesh/Desktop/work/google/wpdev/public/content/plugins/performance/node_modules/commander/lib/command.js:1283:27)
    at /home/thelovekesh/Desktop/work/google/wpdev/public/content/plugins/performance/node_modules/commander/lib/command.js:1081:27
    at Command._chainOrCall (/home/thelovekesh/Desktop/work/google/wpdev/public/content/plugins/performance/node_modules/commander/lib/command.js:1177:12)
    at Command._dispatchSubcommand (/home/thelovekesh/Desktop/work/google/wpdev/public/content/plugins/performance/node_modules/commander/lib/command.js:1077:23)
    at Command._parseCommand (/home/thelovekesh/Desktop/work/google/wpdev/public/content/plugins/performance/node_modules/commander/lib/command.js:1248:19)
```
</details>

Which I don't think is handled by the CLI tool as per its API docs - https://www.npmjs.com/package/commander#display-error

Also, the message it's logging is pretty verbose + CLI doesn't need to handle anything after it, so exiting here shouldn't be an issue or we handle it the CLI way i.e. using `program.error()`.

Reply:
Humm. So when an an exception is thrown it should actually set an error `exitCode` already:

https://github.com/WordPress/performance/blob/9d0a86975a685bdde48bf640ca2f85495d994273/bin/plugin/cli.js#L15-L24

But that's not working?

Reply:
Ohh seems like I missed this. The exit code is 1 yes. Are we ok with a stack trace along with an error message? UX-wise, it can confuse someone with a CLI error while a simple error message in red color shows user has passed the wrong argument.

Reply:
Can we please open a separate issue or PR to discuss this?

This PR is for adding the `since` enhancement, so we should simply throw an error here as it works elsewhere. If the way the CLI tool handles errors has some issues, I'd be supportive of improving that. But let's handle that as its own problem.

Reply:
I see the problem. The issue is that it was doing `console.error( error )` instead of `console.error( error.message )`. I've fixed that and added the red color in b65b8a8d.

Reply:
![image](https://github.com/WordPress/performance/assets/134745/3e6e74d9-e0fd-4177-a011-bc696f3ccf81)